syntax = "proto3";

package order_management;

service OrderManagement {
  rpc CreateOrder (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
  string product_name = 1;
  double price = 2;
  string payment_gateway = 3;
  string card_brand = 4;
  string bank = 5;
  string shipping_address = 6;
  string email = 7;
}

message OrderResponse {
  string order_id = 1;
  string status = 2;
}
